bundles/com.salesforce.bazel.eclipse.core/src/com/salesforce/bazel/eclipse/core/util/trace/Trace.java

```diff
@@ -43,7 +44,7 @@ public static final class Span {
             if (name.isBlank()) {
                 throw new IllegalArgumentException("A blank name is not allowed!");
             }
-
+            startTime = System.currentTimeMillis();
```

Comment:
If it's not an Instant then please add a unit indicator, eg. `startTimeEpocMilli`